### PR TITLE
[db2] Fix stream api example in db2 doc.

### PR DESCRIPTION
**Pull request by GOODBOY008**
Fix db2 doc.

### CODE REVIEW

**Comment by GOODBOY008** [comment]
@ruanhang1993 Pls cc , if you are available.

File: docs/content/connectors/db2-cdc.md [hunk 1]
```diff
@@ -50,8 +50,8 @@ Notes
 
 ###  Not support BOOLEAN type in SQL Replication on Db2
 
-Only snapshots can be taken from tables with BOOLEAN type columns. Currently SQL Replication on Db2 does not support BOOLEAN, so Debezium can not perform CDC on those tables.
-Consider using a different type.
+Only snapshots can be taken from tables with BOOLEAN type columns. Currently, SQL Replication on Db2 does not support BOOLEAN, so Debezium can not perform CDC on those tables.
```
Comment (ruanhang1993) [inline docs/content/connectors/db2-cdc.md:53]: This comma is useless here.

File: docs/content/connectors/db2-cdc.md [hunk 2]
```diff
@@ -203,36 +203,36 @@ _Note: the mechanism of `scan.startup.mode` option relying on Debezium's `snapsh
...
-      .fromSource(db2Source, WatermarkStrategy.noWatermarks(), "Db2 Source")
-      // set 4 parallel source tasks
-      .setParallelism(1)
-      .print().setParallelism(1); // use parallelism 1 for sink to keep message ordering
+    env.addSource(db2Source)
+            .print()
+            .setParallelism(1); // use parallelism 1 for sink to keep message ordering
 
-    env.execute("Print Db2 Snapshot + Binlog");
+    env.execute();
```
Comment (ruanhang1993) [inline docs/content/connectors/db2-cdc.md:235]: We should better provide a meaningful job name.

**Review by ruanhang1993 — COMMENTED** [review]
Thanks for the fix, @GOODBOY008. Left a bit of comments.

File: docs/content/connectors/db2-cdc.md (hunk 1 above)
Comment (GOODBOY008) [reply docs/content/connectors/db2-cdc.md:53]: Romve comma about `BOOLEAN` type not supported?

**Review by ruanhang1993 — APPROVED** [review]
LGTM. Thanks for the contribution.

**Pull request merged**